Logger.md

```diff
@@ -107,6 +115,14 @@ public class DeviceConsoleLogger: Logger {
             Swift.print("[\(levelStringRepresentation(of: level))], {\(message)}")
         }
     }    
+
+    public func log(key: String, value: Any?) {
+        if let value = value {
+            log(level: .info, tag: "KEY", message: "\(key): \(value)")
```

Comment:
Just a question here. Is \(value) equivalent to the old ObjC value.description? 

I did a few tests and looks ok with simple structs but I don't know what happens with private classes.

Reply:
If the type of `value` implements `CustomStringConvertible`, then yes, as it prints the `value.description`. If not, then it just prints the name of the class or the name of the struct + its visible attributes.

I don't get your question about private classes: if are private (of another module), then you can't have instnaces of them, right?